modules/apps/commerce/commerce-currency-service/src/main/java/com/liferay/commerce/currency/service/impl/CommerceCurrencyLocalServiceImpl.java

```diff
@@ -459,6 +437,24 @@ public void updateExchangeRates() throws PortalException {
 			ArrayUtil.toLongArray(commerceCurrencyFinder.getCompanyIds()));
 	}
 
+	@Activate
+	protected void activate(BundleContext bundleContext) {
+		super.setAopProxy(commerceCurrencyLocalService);
```

Comment:
@JanisZhang Usually we don't call `super.setAopProxy` in `activate`. Did you add it in order to make the service registration work below? If so, please convert the `PortalInstanceLifecycleListenerImpl` an OSGi component referencing this local service, for example, `com.liferay.commerce.account.internal.model.listener.PortalInstanceLifecycleListenerImpl`.